ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -588,6 +588,16 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         }
     }
 
+	@objc func handleEscapeKey() {
+		dismissPresentingViewController(animated: true)
+	}
+
+	open override var keyCommands: [UIKeyCommand]? {
+		return [
+			UIKeyCommand(input: UIKeyCommand.inputEscape, modifierFlags: [], action: #selector(self.handleEscapeKey))
```

Comment:
Looks like none of our other Fluent controls explicitly handle UIKeyCommand.inputEscape (or any other UIKeCommands). I will do a little digging around, but I wonder if this is the right fix. It works, but is it a bandaid? Do our other controls have the same problem? If just the DrawerController, it'd be good to understand why it differs from the other controls

Reply:
This is in response to an accessibility bug we go in Loop iOS. Instead of a patch fix just on our end I figured it would be worth it to fix it for Fluent all up as I was able to repro the issue in the test app. Other native Apple controls that pop out a menu do support dismissal with the Escape key (you can see an example in the Notes all), but I suspect since a ViewController has various methods of presentation possible there was no default behaviour.

I did not look into other Fluent controls as I felt that was out of scope for the change.

Reply:
From side conversation, looks like Tab B is the default method of dismissing popovers on iOS with Full Keyboard Access (ESC doesn't actually work for me with Full Keyboard Access in other apps). Are you seeing that doesn't work in the Fluent Demo app?
![image](https://github.com/microsoft/fluentui-apple/assets/49537028/72aa337d-3090-4c96-97ca-ef676b6f2a10)

Reply:
Closing this PR based on conversation above. Tab B can be used as well as overridden by a user to use Esc if that is their preference